### PR TITLE
adding windows functionality

### PR DESCRIPTION
**Pull request by omarcr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +59,22 @@ First, clone or fork this repository. Then to set up your virtual environment, d
...
-4. To add your secret key: create a file anywhere on your computer called `openai.cfg` with the contents `OPENAI_KEY=$YOUR_SECRET_KEY`, where `$YOUR_SECRET_KEY` looks something like `'sk-somerandomcharacters'`. If you are unsure what your secret key is, navigate to the [API docs](https://beta.openai.com/developer-quickstart) and copy the token displayed next to the "secret" key type.
+4. To add your secret key: create a file anywhere on your computer called `openai.cfg` with the contents `OPENAI_KEY=$YOUR_SECRET_KEY`, where `$YOUR_SECRET_KEY` looks something like `'sk-somerandomcharacters'` (including quotes). If you are unsure what your secret key is, navigate to the [API docs](https://beta.openai.com/developer-quickstart) and copy the token displayed next to the "secret" key type.
 5. Set your environment variable to read the secret key: run `export OPENAI_CONFIG=/path/to/config/openai.cfg`
 6. Run `yarn install` in the root directory
 
-To verify that your environment is set up properly, run one of the 3 scripts in the `examples` directory:
+instructions for Windows users:
+1. Create a virtual environment in the root directory: `python -m venv ENV_NAME`
```
Comment (shreyashankar) [inline README.md:67]: s/instructions/Instructions

Please also include a heading for the MacOS or Unix steps!

File: README.md [hunk 2]
```diff
@@ -59,12 +59,22 @@ First, clone or fork this repository. Then to set up your virtual environment, d
...
+2. Activate the virtual environment: ` .\ENV_NAME\Scripts\activate`
+3. Install requirements: `pip install -r api/requirements.txt`
+4. To add your secret key: create a file anywhere on your computer called `openai.cfg` with the contents `OPENAI_KEY=$YOUR_SECRET_KEY`, where `$YOUR_SECRET_KEY` looks something like `'sk-somerandomcharacters'` (including quotes). If you are unsure what your secret key is, navigate to the [API docs](https://beta.openai.com/developer-quickstart) and copy the token displayed next to the "secret" key type.
+5. Set your environment variable to read the secret key: run `set OPENAI_CONFIG=/path/to/config/openai.cfg`
+6. Run `yarn install` in the root directory
 
+To run the demos, modify the following line inside the demos:
```
Comment (shreyashankar) [inline README.md:74]: s/the demos/`api/run_web_app.py`

**Comment by shreyashankar** [comment]
I have some time now -- will merge and make the few small edits :)

**Comment by omarcr** [comment]
> I have some time now -- will merge and make the few small edits :)

Thanks, looks great!

**Pull request merged**